src/main/play-doc/operation/Install.md

```diff
@@ -179,12 +183,27 @@ To increase the verbosity of the ConductR Core logging you can use this command:
 [172.17.0.1]$ echo -Dakka.loglevel=debug | sudo tee -a /usr/share/conductr/conf/conductr.ini
 ```
 
+As with all `conductr.ini` file changes, you must restart the ConductR Core service for this change to take effect.
+
+```bash
+[172.17.0.1]$sudo service conductr restart
+
+```
+
 Similarly for ConductR Agent:
 
 ```bash
 [172.17.0.1]$ echo -Dakka.loglevel=debug | sudo tee -a /usr/share/conductr-agent/conf/conductr-agent.ini
 ```
 
+```bash
+[172.17.0.1]$sudo service conductr-agent restart
+
+```
+
+Please note that Elasticsearch is not required however `conduct logs` and `conduct events` does require
+ Elasticsearch as described in [[Consolidated logging|ConsolidatedLogging]].
+
```

Comment:
I think this might read better if this note was incorporated into the paragraphs at the beginning, around L173-176.